contracts/libs/CBOR.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity >=0.7.0 <0.9.0;
 pragma experimental ABIEncoderV2;
 
 import "./BufferLib.sol";
-
+import "../exports/WitnetTypes.sol";
```

Comment:
Same as the `Buffer` comment, `Value` is a type that belongs to the `CBOR` library. It doesn't make sense to import it from somewhere else.

Reply:
Same as before.